dicom_utils/container/output.py

```diff
@@ -104,7 +104,7 @@ def write(self, collection: RecordCollection, dest: Path) -> RecordCollection:
         return result
 
 
-@OUTPUT_REGISTRY(name="longitudinal", subdir="cases", derived=True)
+# @OUTPUT_REGISTRY(name="longitudinal", subdir="cases", derived=True)
```

Comment:
Should this be removed?

Reply:
I added a remark that this output handler was how we originally handled longitudinal data and that a new organizational system has replaced it. I don't see the need to keep it registered if we plan to remove it in the future.